products/sle/main.pm

```diff
@@ -677,8 +677,7 @@ if (is_jeos) {
 
 # load the tests in the right order
 if (is_kernel_test()) {
-    if (get_var('LTP_BAREMETAL')) {
-        set_var('ADDONURL', 'sdk') if is_sle('=12-SP4');
+    if (get_var('LTP_BAREMETAL') && get_var('INSTALL_LTP')) {
```

Comment:
Are you sure that sdk for SLE12 SP4 is not needed?

Reply:
I am installing SDK trough SCC_ADDONS=sdk in the medium definition.

Reply:
Could you please set it in the code for anything < SLE15? This settings in testsuite breaks SLE15 and we don't want to have specific test suite for SLE15.

Reply:
I do not understand how this change should affect SLE15? As you can see in http://panigale.suse.cz/tests/1337#step/scc_registration/17 the installation works fine in SLE15 with SDK enabled.

The SCC_ADDONS variable is in the medium not in the test suite.

Reply:
Moving talk bellow (not to talk about this on 2 places).